runmariadb.sh

```diff
@@ -12,6 +12,38 @@ if [ ! -d "${DATADIR}/mysql" ]; then
     crudini --set "$MARIADB_CONF_FILE" mysqld innodb_buffer_pool_size 5M
     crudini --set "$MARIADB_CONF_FILE" mysqld innodb_log_buffer_size 512K
 
+    # Config MariaDB to enable TLS
+    if [[  ${IRONIC_TLS_SETUP} == "true" ]]
```

Comment:
same thing here. and the check on the files could be done when setting the mariadb specific variable.